### PR TITLE
add makefile and signature

### PR DESCRIPTION
**Pull request by mrow4a**
@DeepDiver1975 is it correct?

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	--exclude="../$(app_name)/*.log" \
+	--exclude="../$(app_name)/js/*.log" \
+
+# Builds the source package for the app store, ignores php and js tests
+.PHONY: appstore
+appstore:
+	rm -rf $(appstore_build_directory)
+	mkdir -p $(appstore_package_name)
+	cp --parents -r \
+	3rdparty \
```
Comment (DeepDiver1975) [inline Makefile]: no 3rdparty

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	appinfo \
+	controller \
```
Comment (DeepDiver1975) [inline Makefile]: no controller

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+	css \
+	img \
+	js \
+	l10n \
+	lib \
+	rules \
```
Comment (DeepDiver1975) [inline Makefile]: just fix this list please - thx

File: appinfo/signature.json [hunk 4]
```diff
@@ -0,0 +1,132 @@
+{
+    "hashes": {
+        ".git\/COMMIT_EDITMSG": "b51bafa026d00fc2b4dd7f130e7bcc88603db8505b4ebd02666488b2da6f472e094e5acc472b15e2970d39a528f23a8dcd436c7ecd802580e6d58474330c0669",
```
Comment (DeepDiver1975) [inline appinfo/signature.json]: whot?

File: appinfo/signature.json [hunk 5]
```diff
@@ -0,0 +1,132 @@
+{
+    "hashes": {
```
Comment (DeepDiver1975) [inline appinfo/signature.json]: signature not to be added to git

**Comment by mrow4a** [comment]
@DeepDiver1975 done

File: Makefile [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+appstore:
+	rm -rf $(appstore_build_directory)
+	mkdir -p $(appstore_package_name)
+	cp --parents -r \
+	appinfo \
+	css \
+	img \
+	js \
+	l10n \
+	lib \
```
Comment (DeepDiver1975) [inline Makefile]: no templates?

please build the package and test - thx

**Comment by mrow4a** [comment]
@DeepDiver1975 ok it seems to work.

**Comment by mrow4a** [comment]
@pmaier1 I added a screenshot here

**Comment by pmaier1** [comment]
> @pmaier1 I added a screenshot here

Thanks! Still we need to check if it fits to the dimensions required by the marketplace. @hurradieweltgehtunter

**Comment by mrow4a** [comment]
@pmaier1 @hurradieweltgehtunter you are free to change the first image and add screenshots.

**Comment by mrow4a** [comment]
@DeepDiver1975 can we merge this one?

**Comment by hurradieweltgehtunter** [comment]
Updated Screenshot and info.xml. G2g from my side

**Comment by mrow4a** [comment]
and the CLA?

**Comment by hurradieweltgehtunter** [comment]
just done :)

**Pull request merged**